modules/likes/jetpack-likes-settings.php

```diff
@@ -578,6 +613,20 @@ function admin_settings_callback() {
 				delete_option( 'disabled_reblogs' );
 				break;
 		}
+
+		// WPCOM only: Comment Likes
+		if( ! $this->in_jetpack ) {
+			$new_comments_state = !empty( $_POST['jetpack_comment_likes_enabled'] ) ? $_POST['jetpack_comment_likes_enabled'] : false;
```

Comment:
Not a blocker either, but there is a missing space before `empty` that doesn't make my editor happy. :)

Reply:
There were several of these, I fixed 'em. Shouldn't these things get linted? 🤔

Reply:
They should, but only in your editor (if you have phpcs set up). We do not lint PHP on commit (yet), although we have had that discussion in the past.